src/default-settings.js

```diff
@@ -1,167 +1,177 @@
+    ,id_col: 'USUBJID'
+    ,rfendt_col: 'RFENDTC'
+    ,seq_col: 'AESEQ'
+    ,sev_col: 'AESEV'
+    ,sev_vals:
+        ['MILD'
+        ,'MODERATE'
+        ,'SEVERE']
+    ,ser_col: 'AESER'
+    ,term_col: 'AETERM'
+    ,filter_cols: []
+    ,detail_cols: []
+
+  //Standard chart settings
+    ,x: {column: 'wc_value'
+        ,type: 'linear'
+        ,label: null}
+    ,y: {column: null // set in syncSettings()
+        ,type: 'ordinal'
+        ,label: ''
+        ,sort: 'earliest'
+        ,behavior: 'flex'}
+    ,marks: 
+        [
+            {type: 'line'
+            ,per: null // set in syncSettings()
+            ,tooltip: null // set in syncSettings()
+            ,attributes:
+                {'stroke-width': 5
+                ,'stroke-opacity': .5}}
+        ,
+            {type: 'circle'
+            ,per: null // set in syncSettings()
+            ,tooltip: null
+            ,attributes:
+                {'fill-opacity': .5
+                ,'stroke-opacity': .5}} // set in syncSettings()
+        ,
+            {type: 'line'
+            ,per: null // set in syncSettings()
+            ,values: {} // set in syncSettings()
+            ,tooltip: null // set in syncSettings()
+            ,attributes:
+                {'class': 'serious'
+                ,'stroke': 'black'
+                ,'stroke-width': 2}}
+        ,
+            {type: 'circle'
+            ,per: null // set in syncSettings()
+            ,values: {} // set in syncSettings()
+            ,tooltip: null // set in syncSettings()
+            ,attributes:
+                {'class': 'serious'
+                ,'fill': 'none'
+                ,'stroke': 'black'
+                ,'stroke-width': 2}}
+        ]
+    ,legend:
+        {location: 'top'
+        ,label: 'Severity'}
+    ,color_by: null // set in syncSettings()
+    ,colors:
+        ['#66bd63'
+        ,'#fdae61'
+        ,'#d73027']
+    ,date_format: '%Y-%m-%d'
+    ,y_behavior: 'flex'
+    ,gridlines: 'y'
+    ,no_text_size: false
+    ,range_band: 15
+    ,margin: {top: 50}
+    ,resizable: true
 };
 
-export function syncSettings(preSettings){
+export function syncSettings(preSettings) {
     const nextSettings = Object.create(preSettings);
+
+    if (!nextSettings.filter_cols || nextSettings.filter_cols.length === 0)
+        nextSettings.filter_cols =
+            [nextSettings.ser_col
+            ,nextSettings.sev_col
+            ,nextSettings.id_col
+            ,'Participant Status'];
+
     nextSettings.y.column = nextSettings.id_col;
+
     nextSettings.marks[0].per = [nextSettings.id_col, nextSettings.seq_col];
-    nextSettings.marks[0].tooltip = `System Organ Class: [${nextSettings.soc_col}]\nPreferred Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`;
+    nextSettings.marks[0].tooltip = `Verbatim Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`
```

Comment:
It would be nice to be able plot uncoded adverse events, such as early on in the study lifecycle.  Alternatively we could modify the tooltip when the user specifies coding terms.